poc_iot_verifier/src/main.rs

```diff
@@ -1,38 +1,85 @@
-use poc_iot_verifier::{loader, mk_db_pool, purger, runner, Result};
+use clap::Parser;
+use poc_iot_verifier::{loader, purger, runner, Result, Settings};
+use std::path;
 use tokio::signal;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 
+#[derive(Debug, clap::Parser)]
+#[clap(version = env!("CARGO_PKG_VERSION"))]
+#[clap(about = "Helium POC IOT Verifier")]
```

Comment:
This value should appear in Cargo.toml for one place to maintain.
```suggestion
#[clap(about = env!("CARGO_PKG_DESCRIPTION"))]
```
See https://doc.rust-lang.org/cargo/reference/environment-variables.html